tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/search_panel.py

```diff
@@ -101,7 +101,7 @@ def tap(self):
 
             self.root_element.tap()
             # Wait for the displayed app to be that we have tapped
-            Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == app_name)
+            self.wait_to_be_displayed()
```

Comment:
Don't change this one, please. This doesn't apply here, we should fix this in some other way.